src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -448,6 +448,7 @@ ov::Any Plugin::get_ro_property(const std::string& name, const ov::AnyMap& optio
             RW_property(ov::value_cache_precision.name()),
             RW_property(ov::key_cache_group_size.name()),
             RW_property(ov::value_cache_group_size.name()),
+            RW_property(ov::weights_path.name())
```

Comment:
not sure that such property can be got via `core.get_property(CPU, weights_path)`, so `WO_property` is more suitable

Reply:
Done